saleor/plugins/webhook/plugin.py

```diff
@@ -211,12 +211,20 @@ def checkout_updated(self, checkout: "Checkout", previous_value: Any) -> Any:
             WebhookEventType.CHECKOUT_UPDATED, checkout_data
         )
 
-    def notify(self, event: "NotifyEventType", payload: dict, previous_value) -> Any:
+    def notify(self, event: NotifyEventType, payload: dict, previous_value) -> Any:
```

Comment:
I beleive, it should be a `Union[NotifyEventType, str]` - Saleor's events + dynamic event [str]